aws/akka-cloud-deploy.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
code-workspace likely should not be committed?

Reply:
fixed